site/src/pages/RoadmapPage/transfers/UncategorizedCreditsSection.tsx

```diff
@@ -32,7 +46,11 @@ const UncategorizedCreditsSection: FC = () => {
       </SectionDescription>
 
       {courses.map((course) => (
-        <UncategorizedMenuTile key={`${course.name}-${course.units}`} name={course.name} units={course.units} />
+        <UncategorizedMenuTile
+          key={`${course.name}-${course.units}`}
+          uncategorizedInfo={course}
```

Comment:
Is there a reason this can't be kept as two props? If it's for consistency with course transfers, I'd recommend changing the other one instead so the component's code can be nicer